src/main/java/mate/academy/Main.java

```diff
@@ -6,15 +6,19 @@
 import mate.academy.service.AuthenticationServiceImpl;
 import mate.academy.service.OrderService;
 import mate.academy.service.OrderServiceImpl;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class Main {
+    public static final Logger logger = LogManager.getLogger(AuthenticationServiceImpl.class);
```

Comment:
The logger is incorrectly associated with the AuthenticationServiceImpl class. It should be associated with the Main class since that's where it is being used.